ee/src/main/java/org/jboss/as/ee/concurrent/ControlPointUtils.java

```diff
@@ -212,12 +215,18 @@ public T call() throws Exception {
                     if (controlPoint.beginRequest() == RunResult.RUN) {
                         try {
                             return callable.call();
+                        } catch (Exception e) {
+                            throw new ExecutionException(e);
                         } finally {
                             controlPoint.requestComplete();
                         }
                     }
+                } catch(ExecutionException e) {
```

Comment:
Formatting inconsistency - use whitespace to better distinguish betwee control statements and method calls.